### PR TITLE
Use global jjb maven release job

### PR DESCRIPTION
**Pull request by JPWKU**
Have set trigger to build daily to
staging repo for device-bacnet.

Signed-off-by: Jeremy Phelps <jphelps@linuxfoundation.org>

### CODE REVIEW

File: jjb/defaults.yaml [hunk 1]
```diff
@@ -48,3 +48,6 @@
         -Djenkins
         -Dmaven.repo.local=/tmp/r
         -Dorg.ops4j.pax.url.mvn.localRepository=/tmp/r
+
+    # Nexus
+  staging-profile-id: a004df83a6249
```
Comment (zxiiro) [inline jjb/defaults.yaml]: Not sure if you mean to give every project the same staging-profile-id so thought I'd ask. In OpenDaylight this is a project level setting and every project has their own id.

File: jjb/device/device-bacnet.yaml [hunk 2]
```diff
@@ -19,5 +19,7 @@
       - '{project-name}-{stream}-merge-java'
       - '{project-name}-{stream}-verify-docker'
       - '{project-name}-{stream}-merge-docker'
-      - '{project-name}-{stream}-release-version-java-daily-no-sonar'
+      - '{project-name} github-maven-jobs':
+          triggers:
+            - timed: 'H 11 * * *'
```
Comment (zxiiro) [inline jjb/device/device-bacnet.yaml]: "triggers" is not a variable you have to use an actual variable here to override the trigger. Hmm now that I think about it I'm not sure how we can override for  github maybe we have to introduce a new variable.

and i'm not sure if it works to set something underneath a job template  (maybe it does, worth a try)

File: jjb/defaults.yaml (hunk 1 above)
Comment (JPWKU) [reply jjb/defaults.yaml]: for now EdgeX is using the same profile...we will revisit after Barcelona release.

File: jjb/device/device-bacnet.yaml (hunk 2 above)
Comment (JPWKU) [reply jjb/device/device-bacnet.yaml]: I'm removing this now that we have it parameterized in global-jjb.

Comment (tykeal) [reply jjb/device/device-bacnet.yaml]: @zxiiro on your comment about doing an override under a job template. I know that it works, Open-O used to and ONAP does use that pattern extensively.

**Comment by tykeal** [comment]
@JPWKU looks like the change needs a rebase, plus two jobs that need to be re-triggered / run.

**Comment by JPWKU** [comment]
@tykeal  rebased

**Comment by JPWKU** [comment]
ok now to test in sandbox

File: jjb/defaults.yaml [hunk 3]
```diff
@@ -48,3 +48,7 @@
         -Djenkins
         -Dmaven.repo.local=/tmp/r
         -Dorg.ops4j.pax.url.mvn.localRepository=/tmp/r
+
+    # Nexus
+    staging-profile-id: a004df83a6249
+    mvn-staging-id: staging
```
Comment (tykeal) [inline jjb/defaults.yaml]: Interestingly, you're missing the final newline at the end of the file and GitHub is calling you out on it.

Comment (JPWKU) [reply jjb/defaults.yaml]: heh fixing

**Review by tykeal — APPROVED** [review]

**Pull request merged**